test/controllers/ViewTransfersControllerSpec.scala

```diff
@@ -30,13 +30,66 @@ class ViewTransfersControllerSpec extends FrontEndTestHelper {
   val checkPageForStaticElements = new CheckPageForStaticElements
 
   private val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss")
+  private val defaultCurrentStatus = CurrentStatus(None, None, None, None, None, None)
```

Comment:
Naming the arguments here might help with understanding what the statuses are i.e.
`CurrentStatus(series = None, transferAgreement = None, upload = None, clientChecks = None, confirmTransfer = None, export = None)` There are other places with this too but feel free to ignore as they look weird when formatted.